src/GraphQL.Tests/GraphQLSerializersTestData.cs

```diff
@@ -3,16 +3,17 @@
 
 namespace GraphQL.Tests
 {
-    public class DocumentWritersTestData : IEnumerable<object[]>
+    public class GraphQLSerializersTestData : IEnumerable<object[]>
     {
-        public static readonly List<IDocumentWriter> AllWriters = new List<IDocumentWriter>
+        public static readonly List<IGraphQLTextSerializer> AllWriters = new List<IGraphQLTextSerializer>
         {
-            new SystemTextJson.DocumentWriter(new System.Text.Json.JsonSerializerOptions
+            new SystemTextJson.GraphQLSerializer(new System.Text.Json.JsonSerializerOptions
             {
                 WriteIndented = true,
+                PropertyNameCaseInsensitive = true,
```

Comment:
?

Reply:
To make both uniform for purposes of testing.  `Newtonsoft.Json` is always insensitive and I don't think it can be made to be case-sensitive.  Changing this value (currently) only affects the tests in the `GraphQL.Tests.Serialization` namespace.

Reply:
ok